src/users/index.ts

```diff
@@ -34,6 +34,11 @@ interface UsersBase {
 }
 
 export interface Users extends UsersBase {
+  /** @deprecated: Use `users.db.assignRoles` or `users.oidc.assignRoles` instead. */
+  assignRoles: (roleNames: string | string[], userId: string) => Promise<void>;
+  /** @deprecated: Use `users.db.revokeRoles` or `users.oidc.revokeRoles` instead. */
+  revokeRoles: (roleNames: string | string[], userId: string) => Promise<void>;
```

Comment:
getAssignedRoles is also deprecated

Reply:
addressed in https://github.com/weaviate/typescript-client/pull/285/commits/15a56a48fa0493d2985d8242844c07b598be3891